test/module/libs/common/result_test.cpp

```diff
@@ -24,16 +24,28 @@ using iroha::expected::Value;
 using iroha::expected::makeError;
 using iroha::expected::makeValue;
 
+constexpr auto error_case_message = "Unexpected error case";
```

Comment:
maybe `kErrorCaseMsg`?

Reply:
I do not like the idea of putting k in constant names, it is type information, and names should be type-independent

Reply:
It's not about the type, it semantic info

```cpp
#define "asd"
const char *kAsd = "asd";
constexpr auto kAsd = "asd"s;
```

There's 3 different types and exactly same semantic

Reply:
const is type specifier since int and const int are 2 different types.

Also appending k to const name is not present anywhere in our code base and I think it is up to debate whether or not it should be.